### DISCUSSION TITLE
Can you install Talos on a PI CM3+ module

### DISCUSSION THREAD

**Discussion by Sledux** [siderolabs/talos#5110] [Q&A]
I am trying to replace Hypriot on my Turing PI cluster to see if that fixes some corruptions I get on my modules. I was trying to find a way to install Talos on my PI CM3+ 8GB modules but it looks like it might not be supported. Is there a way?

**Comment by frezbo** [comment]
I think the biggest limitations is the 1Gig RAM limit to run control plane nodes

**Comment by kevinelliott** [comment]
I have three of these Turing Pi 1 clusters with CM3+ 8GB modules with 32GB of eMMC. I'd very much like to use these clusters in the home lab. I managed to write the generic RPI image to these modules, but then when I boot the Turing Pi 1, the modules never seem to hit ethernet to contact the Omni. Any ideas?

**Comment by kevinelliott** [comment]
Any chance for a solution here? I was able to flash my nodes, but then they don't ever seem to boot.
